### DISCUSSION TITLE
stoicism/mentors/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#2] [General]
# stoicism/mentors/

https://shanenull.com/stoicism/mentors/

**Comment by shane0** [comment]
mentors

**Reply by shane0** [reply]
# enio morricone

**Reply by shane0** [reply]
The Ecstasy of Gold - Ennio Morricone ( The Good, the Bad and the Ugly ) [High Quality Audio]

**Reply by shane0** [reply]
# tom gilmore

- https://shanenull.com/buddhism/2021/docs_2020/books/tom-gilmore/

**Reply by shane0** [reply]
good bad ugly
https://youtu.be/PYI09PMNazw

**Comment by shane0** [comment]
# ran prieur

https://www.ranprieur.com/

craving the slow collapse 13 years ago saw this coming!

https://vimeo.com/19387422

**Reply by shane0** [reply]
ran prieur

**Reply by shane0** [reply]
he mentions vine
https://en.wikipedia.org/wiki/Vine_Deloria_Jr.

**Reply by shane0** [reply]
# derrick jensen

- not identifying with everything & drinking from a stream
- https://vimeo.com/19372536
- https://vimeo.com/19387376

**Comment by shane0** [comment]
# bukowski

"We're all going to die, all of us, what a circus! That alone should make us love each other but it doesn't. We are terrorized and flattened by trivialities, we are eaten up by nothing."
- Charles Bukowski

**Reply by shane0** [reply]
Style
https://youtu.be/6HpgtW8tGfA?si=nC-gPDq0Q3nZC2kG

**Comment by shane0** [comment]
Anti mentors
Fearless technical documentation

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2022/teachers/

**Comment by shane0** [comment]
# vandana shiva

- https://youtu.be/t8t3I3grHog

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2021/docs_2020/books/tom-gilmore/

**Comment by shane0** [comment]
https://billmckibben.com/

**Comment by shane0** [comment]
https://thirdact.org/get-involved/#fastaction-signup

**Comment by shane0** [comment]
https://en.wikipedia.org/wiki/Herman_Daly

**Reply by shane0** [reply]
William rees

**Reply by shane0** [reply]
Michael dowd

**Reply by shane0** [reply]
William Reese
https://youtu.be/icNvhnFv_wg?si=PVkiB7m644qNscAp

**Comment by shane0** [comment]
marlon brando https://youtu.be/r5IczPLMB7g

**Comment by shane0** [comment]
Elisabeth Kübler-Ross

**Comment by shane0** [comment]
bob marley

**Comment by shane0** [comment]
Marco Pierre white
Gave back his Michelin stars and took his life back

**Reply by shane0** [reply]
https://youtu.be/GSOu3-cymnY

**Comment by shane0** [comment]
Thich nhat hanh 
Transcended the viet nam war

**Comment by shane0** [comment]
Left astrophysics to pursue sustainability
https://youtube.com/@tommurphy2694?si=w7JT6oTYuC1oENBd

**Comment by shane0** [comment]
william r catton bright enough to write overshoot as morons grab

https://youtu.be/ca4S8I9MWw4

**Comment by shane0** [comment]
overshoot read by michael dowd

https://soundcloud.com/michael-dowd-grace-limits

**Comment by shane0** [comment]
# belief

Believing means you do not know, the word requires the premise of not knowing it is dishonesty the fact is you do not know there is no reason to believe it is based on fear of the unknown

Not wanting to know the truth

I know what I know

I do not know what I do not know

Replace belief with honesty and responsibility for unknowns

**Comment by shane0** [comment]
william canton 
https://youtu.be/mt6LwcfjXeM

**Comment by shane0** [comment]
# Linus torvalds

**Reply by shane0** [reply]
“A computer is like air conditioning – it becomes useless when you open Windows.” - Linus Torvalds.

**Comment by shane0** [comment]
https://consciousplanet.org/en

**Comment by shane0** [comment]
ole and maitre errison

https://youtu.be/iCGXVk-cBVk

**Comment by shane0** [comment]
Zappa stories
Demoted David to captain Tom lol
https://youtu.be/0i0Ld6RnOLk?si=vlkPoXGlBHiz5J66

**Comment by shane0** [comment]
# sid smith 

```text
Complex dynamical systems are inherently unpredictable.
They exhibit "mathematical complexity."
```

how to enjoy knowing

https://youtube.com/playlist?list=PLNcGo6a-yKuIubvDb6mIyd0KHQ-7UasJH&si=C4SVeRb-GJFqJSuu

**Reply by shane0** [reply]
https://bsidneysmith.com/writings/essays/all-the-bunnies-in-the-meadow-die

**Reply by shane0** [reply]
ON THE LOOM OF LANGUAGE WE WEAVE OUR PERCEPTIONS INTO THE TAPESTRY OF COMPREHENSION.

**Comment by shane0** [comment]
https://forusa.org/

**Comment by shane0** [comment]
https://thirdact.org/

**Comment by shane0** [comment]
https://plumvillage.org/articles/thich-nhat-hanhs-friendship-with-dr-king#:~:text=A%20year%20later%2C%20on%20June,of%20great%20inspiration%20for%20me.

**Comment by shane0** [comment]
michael reading catton overshoot

https://soundcloud.com/michael-dowd-grace-limits/sets/william-r-catton-jr

**Comment by shane0** [comment]
# anti mentors

- the three poisons: ignorance greed hate 
- anti mentors are unaware of and fully controlled by the three poisons 

## ecocidal warfare

- cultivate perpetual war
- global industrial corporations 
- control nations through economics

### food supply monopoly

- nature cannot be patented so they destroy it
- replacing food with poison corn syrup
- operation ranch hand
- operation trail dust
- salted earth
- carbon pulse
- destroy topsoil

## false philantropy 

- create a crisis forcing the expense on society
- bailouts and subsidies 
- create false solutions geared at recurring crisis and profit 

## **Using Idioms:**

* **Fiddling While Rome Burns:** This idiom refers to someone who is unconcerned about a serious problem while it's happening. It highlights the dissonance between the rich people profiting from the crisis and the rest who suffer.
* **Pulling the Ladder Up After Themselves:** This idiom describes someone who makes it difficult for others to succeed after they have achieved success themselves. It emphasizes how the rich might benefit from the crisis while making it harder for others to recover.
* **A Fox Guarding the Henhouse:** This idiom describes someone who is put in charge of something but has a conflict of interest. It portrays how the rich proposing solutions might prioritize their own benefit over a real fix.

## **Descriptive Phrases:**

* **Privatizing Profits, Socializing Losses:** This phrase directly calls out the strategy of profiting from a crisis while expecting the public to shoulder the burden of fixing it.
* **Manufactured Crises, Manufactured Solutions:** This phrase emphasizes the artificiality of both the crisis and the proposed solutions, suggesting they are created to serve the interests of the wealthy.
* **The Greed Cycle:** This term emphasizes the repetitive nature of the situation, with crises creating opportunities for wealth accumulation, followed by demands for public resources to maintain the status quo.

**Figurative Language:**

* **Vultures Picking at the Carcass:** This metaphor compares the rich to scavengers benefiting from the misfortune of others caused by the crisis.
* **A House of Cards Built on Sand:** This metaphor describes the precariousness of a system that relies on exploiting crises for continued profit.
* **A Band-Aid on a Bullet Wound:** This metaphor highlights the inadequacy of superficial solutions proposed by the rich to address a systemic problem.

**Adding Nuance:**

* You can acknowledge that some crises might be unforeseen, but the response from the wealthy often prioritizes self-preservation over societal well-being.
* You can point out the role of political lobbying and influence in shaping policies that favor the wealthy during crises.

**Comment by shane0** [comment]
https://en.wikipedia.org/wiki/Grigori_Perelman

**Comment by shane0** [comment]
# cormac

- “I WAS AFRAID I WAS GOING TO DIE AND THEN I WAS AFRAID I WASNT"

**Reply by shane0** [reply]
https://shanenull.com/stoicism/cormac/

**Reply by shane0** [reply]
https://shanenull.com/2023/books/cormac/

**Comment by shane0** [comment]
# william rees

- adding a page

https://www.thegreatsimplification.com/episode/53-william-rees

https://mahb.stanford.edu/wp-content/uploads/2023/02/855-Article-Text-4479-1-10-20230131.pdf

**Reply by shane0** [reply]
https://shanenull.com/stoicism/william_rees/

**Reply by shane0** [reply]
https://nssdc.gsfc.nasa.gov/planetary/factsheet/marsfact.html

**Reply by shane0** [reply]
## alan drengson

https://www.youtube.com/watch?v=wjrrww5aIz4

```text
No amount of logical reasoning, legal argumentation, or ethical axiology will solve a problem that is misunderstood.
```

**Comment by shane0** [comment]
# self reliance rights

- Louis Rossman 
https://youtube.com/@rossmanngroup?si=2ZiOXVzaFzWszR7u

**Reply by shane0** [reply]
# engineers who refuse to use technology for profit vs quality

- I search for quality and these temporary pockets of low entropy where strong people refuse to use technology for profit over quality
- cenk & elektron
- dana gleason left dana designs to make quality packs - mystery ranch - now bought by yeti
- Bart Kyzar and Mark Falvai left chrome bags to start mission workshop bags
- Tern bicycles was founded by Florence Shen and Joshua Hon, wife and son of David T. Hon, founder of the established Dahon brand of folding bicycles.

**Comment by shane0** [comment]
# sage of baltimore

https://gettysburgconnection.org/quotable-quotes-from-the-sage-of-baltimore/

**Reply by shane0** [reply]
```text
“On some great and glorious day, the plain folks of the land will reach their heart’s desire at last, and the White House will be adorned by a downright moron.”   - sage of baltimore
```

1920

**Reply by shane0** [reply]
```text
“War will never cease until babies begin to come into the world with larger cerebums and smaller adrenal glands.” - sage of baltimore
```

**Reply by shane0** [reply]
```text
“The whole aim of practical politics is to keep the populace alarmed (and hence clamorous to be led to safety) by an endless series of hobgoblins, most of them imaginary.” - sage of baltimore
```

**Comment by shane0** [comment]
# translators

Here’s a table that summarizes notable English translations of *Prajñāpāramitā* texts, including the translator's name, the dates of translation, and the specific texts they translated:

| Translator            | Date of Translation | Texts Translated                                   | Additional Information                        |
|-----------------------|---------------------|----------------------------------------------------|----------------------------------------------|
| **Edward Conze**      | 1958 (revised 1973) | *Heart Sutra*, *Diamond Sutra*, *Aṣṭasāhasrikā Prajñāpāramitā* | Pioneering translator; emphasized philosophical aspects. |
| **Thich Nhat Hanh**   | 1991                | *Heart Sutra*, *Diamond Sutra*                     | Focused on practical applications in mindfulness and Zen. |
| **Sangharakshita**    | 1999                | *Heart Sutra*                                     | Founder of Triratna Buddhist Community; interpretative approach. |
| **L. Richard Harris**  | 1981                | *Aṣṭasāhasrikā Prajñāpāramitā*                    | Scholarly translation with attention to historical context. |
| **Tenzin Gyatso (14th Dalai Lama)** | 1997 | *The Heart of the Prajñāpāramitā Sutra*           | Focused on the teachings and implications for modern practitioners. |
| **Kazuaki Tanahashi** | 2010                | *The Heart Sutra*                                 | Emphasized artistic expression in the translation. |
| **Geoffrey Shugen Arnold** | 2002           | *The Heart Sutra*                                 | Zen priest; aimed at accessible interpretation for practitioners. |
| **M. T. (Michaela) D'Amato** | 2003      | *Prajñāpāramitā in Eight Thousand Lines*          | Comprehensive English translation of a major *Prajñāpāramitā* text. |

This table captures key translators and their contributions to the understanding of *Prajñāpāramitā* literature in English.

**Reply by shane0** [reply]
# synopsis

- The Large Sutra on Perfect Wisdom with the Divisions of the Abhisamayalankara
- isbn 9780520022409

Some two thousand years ago Buddhism experienced a major reformation through a movement called the Mahayana, or "Great Vehicle," which dominated religious thought in much of Asia for many centuries and still exerts considerable influence. The basic Mahayana texts were sermons ascribed to the Buddha, called "Sutras" in Sanskrit. The earliest and most influential of these Mahayana Sutras had the "perfection of wisdom" as its main subject matter. Of these texts, the famous "Diamond" and "Heart" Sutras have been known in the West for many years, but they are merely condensations of the original "Large Sutra on Perfect Wisdom" that took shape between 50 and 200 A.D. In the present volume, Dr. Conze offers the result of thirty-five years of close study, and makes available this "Large Sutra," the key document for dealing with early Mahayana doctrine. This scripture has, throughout the centuries, been revered as "The great mother of the Buddhas and Bodhisattvas" not only in India but in China, Tibet, Japan, Mongolia and Southeast Asia as well. It is now made available in its complete form for the first time in an annotated translation.

**Comment by shane0** [comment]
# robin

- daily routines
- drink dishwater lol
- wifi at libraries
- walking https://youtu.be/iFFlOkL8D3Y

**Comment by shane0** [comment]
# ernest becker

> The real world is simply too terrible to admit. It tells man that he is a small trembling animal who will someday decay and die. Culture changes all of this, makes man seem important, vital to the universe, immortal in some ways.
>> Ernest Becker

- more quotes https://www.natsnotes.com/p/denial-of-death

**Comment by shane0** [comment]
# complexity

```text
ON THE LOOM OF LANGUAGE WE WEAVE OUR PERCEPTIONS INTO THE TAPESTRY OF COMPREHENSION.
```

```text
Complex dynamical systems are inherently unpredictable.
They exhibit "mathematical complexity."
```

- https://youtu.be/5r3tLh8YkN8?si=mBYcWNmi1HqNZwiV

**Comment by shane0** [comment]
# kurt vonnegut

- https://shanenull.com/2023/books/kurt/

**Comment by shane0** [comment]
# sid smith

- https://playgameoflife.com/lexicon/101

**Reply by shane0** [reply]
https://youtube.com/playlist?list=PLNcGo6a-yKuIubvDb6mIyd0KHQ-7UasJH&si=dB249RtbConvlD5s

**Reply by shane0** [reply]
concerns: enjoying the collapse of this non sustainable culture because the collapse is also the start of building a new sustainable culture

**Comment by shane0** [comment]
# edward witten

- reality is described by laws that are interesting and solvable mathematically
- math is uncanny because its as if the creator of the universe was a mathematician
- the critics don't seem to want to work very hard on a competitive theory or suggest one

**Reply by shane0** [reply]
https://youtu.be/Qje1ak6XF20

**Reply by shane0** [reply]
entropy https://youtu.be/A2EXjl7paBQ

**Comment by shane0** [comment]
Rik mayall

https://youtu.be/mnpdn7ge-jI?si=hqpOZjFxLFx3l-XR

**Comment by shane0** [comment]
# death of the american dream

Several authors and scholars have explored the transformation of the *American Dream* into a global nightmare, touching on themes of capitalism, imperialism, inequality, and environmental destruction. Here are some key thinkers and books on the topic:

### **1. Economic Exploitation & Capitalism**  
- **Chris Hedges** – *Empire of Illusion: The End of Literacy and the Triumph of Spectacle* (2009)  
  - Critiques how American consumerism, corporate greed, and media spectacle have eroded democracy and morality.  
- **Naomi Klein** – *The Shock Doctrine: The Rise of Disaster Capitalism* (2007)  
  - Examines how neoliberal policies have led to global economic exploitation and instability.  
- **Joseph Stiglitz** – *The Price of Inequality* (2012)  
  - Argues that America’s wealth gap is a symptom of a rigged system that threatens global economic stability.  

### **2. U.S. Imperialism & Military Interventions**  
- **Howard Zinn** – *A People’s History of the United States* (1980)  
  - Challenges the myth of American exceptionalism by highlighting exploitation, war, and corporate influence.  
- **Noam Chomsky** – *Hegemony or Survival: America’s Quest for Global Dominance* (2003)  
  - Analyzes U.S. foreign policy as an extension of imperial ambition rather than democratic values.  
- **William Blum** – *Killing Hope: U.S. Military and CIA Interventions Since World War II* (1995)  
  - Documents covert and overt U.S. interventions that have destabilized nations and fueled global conflicts.  

### **3. Environmental Destruction & Climate Crisis**  
- **David Wallace-Wells** – *The Uninhabitable Earth: Life After Warming* (2019)  
  - Shows how American-led industrialization and fossil fuel dependence have accelerated global climate catastrophe.  
- **Naomi Klein** – *This Changes Everything: Capitalism vs. The Climate* (2014)  
  - Argues that corporate-driven capitalism is incompatible with serious climate action.  
- **Rachel Carson** – *Silent Spring* (1962)  
  - One of the first books to expose the environmental costs of unchecked industrial progress.  

### **4. Cultural Domination & Corporate Power**  
- **Jean Baudrillard** – *America* (1986)  
  - A French philosopher’s critical analysis of American consumer culture, media saturation, and excess.  
- **Thomas Frank** – *What's the Matter with Kansas?* (2004)  
  - Explores how cultural and political manipulation keeps working-class Americans voting against their economic interests.  
- **Guy Debord** – *The Society of the Spectacle* (1967)  
  - Though not U.S.-specific, it critiques how capitalism turns life into spectacle, influencing American consumer culture.  

### **5. The Decline of the American Dream**  
- **F. Scott Fitzgerald** – *The Great Gatsby* (1925)  
  - A classic novel exposing the emptiness and corruption of the American Dream.  
- **George Packer** – *The Unwinding: An Inner History of the New America* (2013)  
  - Chronicles the decline of the middle class and the rise of inequality.  
- **Barbara Ehrenreich** – *Nickel and Dimed: On (Not) Getting By in America* (2001)  
  - Investigates the struggles of low-wage workers in a country that claims to offer opportunity.  

### **6. The Illusion of Progress & Social Decay**  
- **Michel Foucault** – *Discipline and Punish* (1975)  
  - Though not about America specifically, it examines how power structures control and oppress populations, relevant to critiques of the U.S. prison-industrial complex.  
- **Edward Said** – *Orientalism* (1978)  
  - Analyzes how Western (especially American) cultural hegemony distorts global perceptions of the "Other."  
- **Arundhati Roy** – *Capitalism: A Ghost Story* (2014)  
  - Critiques how American-led capitalism fuels global exploitation and suffering.  

These authors provide deep, critical insights into how the ideals of the *American Dream*—wealth, freedom, and democracy—have often been subverted into economic, environmental, and geopolitical crises, turning it into a nightmare for much of the world.

**Comment by shane0** [comment]
# frank zappa

I think that any artistic decision that is based on whether or not you are going to make money it is not really an artistic decision. It is a business decision. And there are a lot of things that I can do to earn a living and a lot of things that I have already done to earn a living which produce the amount of capital needed to do this project. I came here to spend money on an English orchestra and record my music, so I can take it home and I can listen to it. And... if somebody else likes that kind of stuff, I will make it available on a record so that they can hear it. That is my part of the public service of spending the money to make this event happen. No foundation grant, no government assistance, no corporation, no comittee. Just a crazy guy who spent the money to hire English musicians to do a concert at the Barbican and make an album for Barking Pumpkin Records.”


"I think that any artistic decision that is based on whether or not you are going to make money is not really an artistic decision. It is a business decision."

While all other guys spent all their money on cars, I bought records (I didn't have a car). I went to second hand stores to buy jukebox records with rhythm and blues songs.


Information is not knowledge.
Knowledge is not wisdom.
Wisdom is not truth.
Truth is not beauty.
Beauty is not love.
Love is not music.
Music is THE BEST

**Reply by shane0** [reply]
- hunter thompson "The music business is a cruel and shallow money trench, a long plastic hallway where thieves and pimps run free, and good men die like dogs. There's also a negative side." Hunter S. Thompson

**Reply by shane0** [reply]
“Definition of rock journalism: People who can't write, doing interviews with people who can't think, in order to prepare articles for people who can't read.”

**Reply by shane0** [reply]
“There is more stupidity than hydrogen in the universe, and it has a longer shelf life.”

**Reply by shane0** [reply]
“The illusion of freedom will continue as long as it's profitable to continue the illusion. At the point where the illusion becomes too expensive to maintain, they will just take down the scenery, they will pull back the curtains, they will move the tables and chairs out of the way and you will see the brick wall at the back of the theater.”
― Frank Zappa

**Reply by shane0** [reply]
Tobacco is my favorite vegetable

**Comment by shane0** [comment]
alice walker

**Comment by shane0** [comment]
# David Graeber

https://youtu.be/Hn78MhPmbCc?si=HwdM-boWDp0jGbWW

**Reply by shane0** [reply]
wish he didn’t die it would have been interesting to hear his commentary on ai slop and the anomie we are in now

**Comment by shane0** [comment]
# Joseph Tainter

 is an American anthropologist and historian.  He is best known for his academic work on societal collapse, specifically his landmark 1988 book, The Collapse of Complex Societies

**Reply by shane0** [reply]
concerns: teaching people to think about the long term future

**Reply by shane0** [reply]
https://youtu.be/caamifTtlsQ?si=fVo-HQOG59NYJ0U8

**Comment by shane0** [comment]
jared diamond

**Reply by shane0** [reply]
guns germs steel
why societies choose to fail

concerns: water and topsoil
